sdk/identity/identity/src/msal/nodeFlows/msalNodeCommon.ts

```diff
@@ -108,6 +133,10 @@ export abstract class MsalNode extends MsalBaseUtilities implements MsalFlow {
   protected azureRegion?: string;
   protected createCachePlugin: (() => Promise<msalNode.ICachePlugin>) | undefined;
   protected createCachePluginCae: (() => Promise<msalNode.ICachePlugin>) | undefined;
+  protected createNativeBrokerPlugin: (() => Promise<msalNode.INativeBrokerPlugin>) | undefined;
+  protected enableMSAPassthrough?: boolean;
```

Comment:
Change casing to `enableMsaPassthrough`.

Reply:
Done